### DISCUSSION TITLE
how to access data of externaly loaded items (select,multiselect) ...

### DISCUSSION THREAD

**Discussion by clanghof** [vueform/vueform#249] [Questions]
Trying to access the items I used form$.value.el$('path.to.my.select').items.
For localy stored items this works fine. 
But I'm loading data for my selects (dynamicly - with params) from an extern ressource and want
to get access to the loaded items. Only path is shown...not the currently avaible data.
How to get?

**Comment by clanghof** [comment]
I've got it out by checking the source...
**Select + MultiSelect**
form$.value.el$('your.path.to.element').resolvedOptions;
**tags**
form$.value.el$('your.path.to.element').input.extendedOptions
**checkboxgroup+radiogroup**
form$.value.el$('your.path.to.element').resolvedOptions

thats a bit confusing because: 
a) depending on the element mode of access is different  - this could be unified
b) the quality of data is different. While select + multiselect delivers just key + value (may be disabled as well not tested yet)
the rest returns the complete json-object stored in items (from my point of view better)

**Reply by adamberecz** [reply]
It's a good point. I've added it to the idea pool: https://github.com/vueform/vueform/discussions/253 Would you be happy to do a PR? This is the file that needs to be touched: https://github.com/vueform/vueform/blob/main/src/composables/elements/useAsyncItems.js
